diesel/src/query_builder/only_clause.rs

```diff
@@ -0,0 +1,85 @@
+//! `ONLY` clause.
+//!
+//! Can only be used on tables, e.g. `my_table::dsl::table.only().select(something)` should generate
+//! something like `SELECT something FROM ONLY my_table`.
+
+use super::{AsQuery, AstPass, Backend, QueryFragment, QueryId, SelectStatement};
+use crate::{
+    associations::HasTable, expression::ValidGrouping, prelude::*, query_dsl::methods::*,
+    query_source::Table,
+};
+
+/// Represents a query with an `ONLY` clause.
+#[derive(Debug, QueryId)]
+pub struct Only<T> {
+    query: T,
+}
+
+/// Add `ONLY` before a query source.
```

Comment:
This is the main entry point for using `from only ...` clauses. I feel the documentation here could be a bit more detailed.